### DISCUSSION TITLE
Second Monitor Taskbar icons does not have right click context menus

### DISCUSSION THREAD

**Discussion by sislamoglu** [valinet/ExplorerPatcher#2727] [General]
In Windows 11 with multiple monitors which has option to show windows which display they are on, when you right click on icons, you don't get menus for the application. You only get standard menu... It does not happen as default behaviour of Windows 11

Display 1: (Main)
![monitor1](https://github.com/valinet/ExplorerPatcher/assets/1365217/81dd5868-9ff7-4eda-86ea-9300800a25aa)

Display 2:
![monitor2](https://github.com/valinet/ExplorerPatcher/assets/1365217/46e7f1b4-20f3-470b-ba60-70f6f0c52a39)

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

Please include any other software you have installed that modifies the UI of windows.

**Reply by sislamoglu** [reply]
23H2 Build 22631.3007

I have PowerToys from Microsoft and app that makes smart zones for my monitor application (Dell Display Manager)

**Reply by sislamoglu** [reply]
Reinstalled Explorer Patch and this has gone, seems it has to do with one of settings I picked before. I can reopen this ticket if I encounter it again, sorry for false flag
